integration/monai/examples/spleen_ct_segmentation_sim/README.md

```diff
@@ -84,40 +84,40 @@ sed -i "s|/workspace/data/Task09_Spleen|${PWD}/data/Task09_Spleen|g" ${JOB_NAME}
 In resource restricted environments where you need to simulate several clients (-n 2 in this case) on the same GPU device,
 you can run the simulator using:
 
-<pre style="background: #f4f4f4; border: 1px solid #ddd; border-left: 3px solid #02a3a3; line-height: 1.6; padding: 1.5em;">
+```
 nvflare simulator job --workspace /tmp/nvflare/sim_spleen_ct_seg --threads 1 --n_clients 2
-</pre>
+```
 
 #### 3.2 Multiple threads, multiple gpus
```

Comment:
no 3.1?

Reply:
There is https://github.com/NVIDIA/NVFlare/blob/ffa774fcdd954a5d4bc27952757f1dac65c68fe7/integration/monai/examples/spleen_ct_segmentation_sim/README.md#31-single-thread-single-gpu

Reply:
oh see it now, auto-collapsed